pacman/README.md

```diff
@@ -0,0 +1,192 @@
+# Pacman
+
+<p align="center">
+<img src="http://github.com/glouppe/info8006-introduction-to-ai/blob/pacman_project/pacman/pacman_game.png" width="50%" />
```

Comment:
Can you update `pacman_project` to `master`? (this will be required after merge)